src/rollup/plugins/storage.ts

```diff
@@ -24,7 +24,6 @@ export function storage(nitro: Nitro) {
   const driverImports = [...new Set(mounts.map((m) => m.driver))];
 
   const bundledStorageCode = `
-import { prefixStorage } from 'unstorage'
```

Comment:
I think we might remove from other place as only prefixStorage usage is now here